### PR TITLE
[CDAP-15787] Add validation apis to hbase plugins

### PR DESCRIPTION
**Pull request by CuriousVini**

### CODE REVIEW

**Comment by CuriousVini** [comment]
build: https://builds.cask.co/browse/HYP-BAD582-4

File: hbase-plugins/src/main/java/io/cdap/plugin/HBaseConfig.java [hunk 1]
```diff
@@ -66,14 +72,26 @@ public HBaseConfig(String referenceName, String tableName, String rowField, @Nul
    * @throws IllegalArgumentException if the schema is null or not as valid JSON
    */
   public Schema getSchema() {
-    if (schema == null) {
-      throw new IllegalArgumentException("Schema cannot be null");
+    if (Strings.isNullOrEmpty(schema)) {
+      throw new IllegalArgumentException("Schema is either null or empty. It must be specified.");
```
Comment (albertshau) [inline hbase-plugins/src/main/java/io/cdap/plugin/HBaseConfig.java]: null or empty is technical. We can just say something like:

Schema must be provided.

**Review by albertshau — APPROVED** [review]
minor comments, otherwise lgtm

**Comment by CuriousVini** [comment]
latest build: https://builds.cask.co/browse/HYP-BAD582-8

**Pull request merged**